### PR TITLE
[Security Solution] Update SentinelOne integration README with info. on configuring response actions

### PR DESCRIPTION
**Pull request by paul-tavares**
## Proposed commit message

- Adds information to the SentinelOne (S1) integration `README` file about the ability for sending Response Actions to S1 hosts
    - Changes goes along with Feature being implemented in Security Solution that supports sending `isolate` and `release` response actions.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- none

## How to test this PR locally

- Start the stack using `--version=8.12.0-SNAPSHOT`:

```shell
elastic-package stack up -v -d --version=8.12.0-SNAPSHOT
```

- Navigate to the SentinelOne integration page
    - Note that the version at the top is `1.20.0`
    - Note the updated integration description that includes a link to Documentation
- Click the `documentation` link
    - You will (until v8.12.0 is GA'd) received a `404` "Page not found". This is expected since v8.12 has not yet been released. The page is accessible however by changing the `/current/` in the URL with `/master/`: image

## Related issues

- none

## Screenshots

image

### CODE REVIEW

File: packages/sentinel_one/changelog.yml [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # newer versions go on top
-- version: "1.19.0"
+- version: "1.20.0"
```
Comment (paul-tavares) [inline packages/sentinel_one/changelog.yml:2]: Am I doing this right? or should this be `1.20.0-next`?

**Review by efd6 — COMMENTED** [review]
I think it would be worth waiting until 8.12 is released before this change is made.

**Comment by paul-tavares** [comment]
Hi @efd6 - thanks for the quick feedback. I'll follow up with our team to see if that is ok. I'm thinking its fine given the nature of the change I'm introducing.

File: packages/sentinel_one/changelog.yml (hunk 1 above)
Comment (kush-elastic) [reply packages/sentinel_one/changelog.yml:2]: It should be `1.20.0` only. the current change is correct.

File: packages/sentinel_one/manifest.yml [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
+version: "1.20.0"
 description: Collect logs from SentinelOne with Elastic Agent.
 type: integration
 categories:
   - security
   - edr_xdr
 conditions:
   kibana:
-    version: ^8.7.1
+    version: ^8.12.0
```
[the remark this answers is not in the document]
Comment (kush-elastic) [reply packages/sentinel_one/manifest.yml:12]: Potential issues may arise with future updates to the integration, potentially limiting the user's scope. Is there an alternative approach available? Is this feature exclusively available in the latest version, `8.12.0`?

Another option is to add a note, suggesting users update their clusters to version `8.12.0` if they wish to utilize this feature. This way, the current integration doesn't necessarily need to be tailored specifically for version `8.12.0`.

[the remark this answers is not in the document]
Comment (kush-elastic) [reply packages/sentinel_one/manifest.yml:12]: The final decision rests with the designated "Code Owners."

**Comment by mrodm** [comment]
Hi @paul-tavares, please update your branch with the latest contents from main branch. There was an important PR merged updating the CI pipelines. Thanks!

File: packages/sentinel_one/manifest.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (paul-tavares) [reply packages/sentinel_one/manifest.yml:12]: Thank you @kush-elastic . I will check with our Product Management team and potentially add an additional sentence that indicates the new capability is only available with v8.12

**Comment by paul-tavares** [comment]
Taking this out of draft as we are approaching the GA for `v8.12.0`.

**Review by joepeeples — APPROVED** [review]
New content LGTM, and I added some extra edits in https://github.com/elastic/integrations/pull/8611/commits/74b731e77de258d99e907d9c17e4b38335e39be7 which have already been incorporated. Thanks!

File: packages/sentinel_one/changelog.yml (hunk 1 above)
Comment (ebeahan) [reply packages/sentinel_one/changelog.yml:2]: Since we're only updating docs, maybe we do a patch release for `1.19.1` instead of a new minor?

Comment (paul-tavares) [reply packages/sentinel_one/changelog.yml:2]: 🤷 
hi @ebeahan - I'm not sure if that is the best way - this is the my first time making such changes. If you believe that is best I can change it. Please let me know

**Review by ebeahan — APPROVED** [review]

**Pull request merged**